edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/DeviceScopeIdentitiesCache.cs

```diff
@@ -152,12 +159,12 @@ await serviceIdentity
                 }
                 else
                 {
-                    Events.SkipRefreshServiceIdentity(id, this.identitiesLastRefreshTime[id], this.refreshDelay);
+                    Events.SkipRefreshServiceIdentity(refreshTarget, this.identitiesLastRefreshTime[refreshTarget], this.refreshDelay);
                 }
             }
             catch (Exception e)
             {
-                Events.ErrorRefreshingCache(e, id);
+                Events.ErrorRefreshingCache(e, refreshTarget);
```

Comment:
log onBehalfOfDevice and invokeServiceIdentitiesUpdated?

Reply:
added log for onBehalfOfDevice, doesn't really make sense to log invokeServiceIdentitiesUpdated.

---
In reply to: [479403827](https://github.com/Azure/iotedge/pull/3467#discussion_r479403827) [](ancestors = 479403827)